packages/components/src/components/ScalarSearchInput/ScalarSearchInput.vue

```diff
@@ -32,7 +32,7 @@ const attrs = computed(() => {
 })
 
 const variants = cva({
-  base: 'flex items-center rounded border bg-b-1/2 text-sm font-medium has-[:focus-visible]:bg-b-1 has-[:focus-visible]:outline',
```

Comment:
I think tailwind might be interpreting it as `bg-b-1` with a transparency of "2" see https://tailwindcss.com/docs/background-color#changing-the-opacity

Did you try making the variable `1-2` instead of `1/2`?

Reply:
it is changing border colors for some reason I'm not sure of, i want it to mix bg-b-1 and bg-b-2